weather_aplication/OpenMeteoJSONDecoded.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+struct OpenMeteoJSONDecoded: Decodable {
```

Comment:
po co słowo JSON? Z perspektywy strucla nie interesuje nas to czy byliśmy dekodowani z JSONA czy tworzeni po prostu zwykłym initem